samples/AspNetCore3xEndpointSample.Web/Controllers/CustomersController.cs

```diff
@@ -1,5 +1,9 @@
-﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
-// Licensed under the MIT License.  See License.txt in the project root for license information.
+//-----------------------------------------------------------------------------
+// <copyright file="CustomersController.cs" company=".NET Foundation">
+//      Copyright (c) .NET Foundation and Contributors. All rights reserved. 
+//      See License.txt in the project root for license information.
```

Comment:
do we have the correct License.txt in the project root folder?

Reply:
I have updated it in the second pr for the non-sources files.